src/gui/tracks/TrackContentWidget.cpp

```diff
@@ -101,22 +125,29 @@ void TrackContentWidget::updateBackground()
 	pmp.fillRect( w, 0, w , h, lighterColor() );
 
 	// draw lines
-	// vertical lines
-	pmp.setPen( QPen( gridColor(), 1 ) );
-	for( float x = 0; x < w * 2; x += ppb )
+	// draw fine grid
+	pmp.setPen( QPen( fineGridColor(), fineGridWidth() ) );
+	for( float x = 0; x < w * 2; x += ppb * fineGridResolution )
+	{
+		pmp.drawLine( QLineF( static_cast<int>( x ), 0.0, static_cast<int>( x ), h ) );
```

Comment:
Why are you casting to `int` if you are using `QLineF`? The advantage of `QLineF` is that you can use non-integer coordinates.

Reply:
Honestly, I'm not sure. I think I left it there on accident. Good catch - I'll remove it.